modules/tensor_mechanics/test/tests/rom_stress_update/gold/3tile_out.csv

```diff
@@ -0,0 +1,20 @@
+time,creep_rate,partition_weight,rhoi_rate,rhom_rate,temperature,vmJ2_in
+0,0,0,0,0,0,25.68
+0.1,2.718281828459,0,-1312930123145.7,-15494206422217,930,25.68
+0.2,2.718281828459,1,-1312930123145.7,-15494206422217,930,45
+0.3,2.718281828459,1,-1312930123145.7,-15494206422217,930,55.28
+0.4,2.718281828459,1,-1312930123145.7,-15494206422217,930,60
+0.5,2.718281828459,1,-1312930123145.7,-15494206422217,930,67.12
+0.6,2.718281828459,1,-1312930123145.7,-15494206422217,930,75
+0.7,2.718281828459,1,-1312930123145.7,-15494206422217,905,75
+0.8,2.718281828459,1,-1312930123145.7,-15494206422217,895,75
+0.9,2.718281828459,1,-1312930123145.7,-15494206422217,881,75
+1,2.718281828459,1,-1312930123145.7,-15494206422217,860,75
+1.1,2.718281828459,1,-1312930123145.7,-15494206422217,821,75
+1.2,2.718281828459,0,-1312930123145.7,-15494206422217,860,55.28
+1.3,2.718281828459,0.45082253523362,-1312930123145.7,-15494206422217,881,60
```

Comment:
You're not doing anything for fractional partition weights. Would it be smart to include that in this test too?

Reply:
Actually, we are probing regions where partitions overlap while also multiple tiles overlap. To see this, check the "partition_weight" column in the gold file 3tile_out.csv. This has been done intentionally to be sure these overlaps don't cause errors.

Reply:
Oh I see it now, there are a couple regions where partition_weight isn't zero or one, I missed it the first time.